metabrainz/templates/index/team.html

```diff
@@ -187,51 +187,6 @@ <h4>{{ _('<span class="name">Nicolas Pelletier</span> &mdash; Software Engineer
 
     <h2>{{ _('Voluntary Project Leaders') }}</h2>
 
-    <div class="person">
-      <img class="pic" src="{{ url_for('static', filename='img/team/ben_ockmore.jpg') }}"/>
```

Comment:
Should probably remove the images from the repository as well, then.

Reply:
Good call, thanks.